cmd/webhook/main.go

```diff
@@ -109,8 +121,11 @@ func main() {
 		Namespace:                       system.Namespace(),
 		Port:                            8443,
 		SecretName:                      "webhook-certs",
-		WebhookName:                     "webhook.serving.knative.dev",
+		ResourceMutationWebhookName:     "resource.webhook.serving.knative.dev",
 		ResourceAdmissionControllerPath: "/",
+		ConfigValidationWebhookName:     "config.webhook.serving.knative.dev",
+		ConfigValidationControllerPath:  "/config-validation",
+		NamespaceLabel:                  "serving.knative.dev/release",
```

Comment:
for kube api < 1.14, the only way to scope the webhook to a namespace is by using the `NamespaceSelector`. Shall we use the above label?

Reply:
What was introduced in 1.14 that's better?  Do you have a link?

Reply:
Also, it would need a 1.14 version of `client-go`?

Reply:
in 1.14 rules can be defined to have a tighter scope for the namespace: https://github.com/kubernetes/api/blob/release-1.14/admissionregistration/v1beta1/types.go#L63

we will still need the NamespaceSelector to tie it only to the serving namespace. and no it wouldn't need a bump to client-go.